src/backend/lima.ts

```diff
@@ -2073,10 +2080,6 @@ CREDFWD_URL='http://${ hostIPAddr }:${ stateInfo.port }'
     return this.client?.listServices(namespace) || [];
   }
 
-  async isServiceReady(namespace: string, service: string): Promise<boolean> {
```

Comment:
Looks like we stopped calling this method some time ago.

Reply:
Yes, which is why I removed it :)